### DISCUSSION TITLE
After enabling new 'Sites' need an easier way to enable the Site for all Sections

### DISCUSSION THREAD

**Discussion by ifrimere** [craftcms/cms#8703] [Ideas]
### Description
In a Craft3 multisite, if a user has multiple Sites enabled (ex. 1 primary source language & 3 target languages) and then adds a 4th new target language, they have to manually go into each Section and enable the new site for that Section.  It would be cool if after adding a new Site, the system offered to automatically enable all Sections for that site (locale).

### Steps to reproduce
1. Add a new Site to a multisite with multiple Sections enabled

### Additional info
1. Several of our Translations plugin customers have forgotten to activate Sections for their newly added Sites.

**Comment by piotrpog** [comment]
I think there should be config file setting for automatic enabling all sections upon adding new site.
